src/main/java/com/kodilla/ecommercee/domain/groups/Group.java

```diff
@@ -12,15 +12,16 @@
 @NoArgsConstructor
 @Getter
 @Setter
-@Entity(name = "groups")
+@Entity
+@Table(name = "\"group\"")
 public class Group {
```

Comment:
Dodałem @Table ze względu z nazwą ze wzgledu na to, ze nie chciały wykonwyać się servicy takie jak count , czy findAll, wpisałem błąd w google i wyszło na to że musze dodać @Table i rzeczywiscie teraz działają nie tylko save i dalete ale reszta rzeczy takie jak count() i findAll() też